README.md

```diff
@@ -142,6 +142,111 @@ Population handling for `EdgeStorage` is analogous to `NodeStorage`:
 >>> selection = population.connecting_edges([1, 2, 3], [4, 5, 6])
 ```
 
+## Reports
+
+### SpikeReader
+```python
+>>> import libsonata
+
+>>> spikes = libsonata.SpikeReader('path/to/H5/file')
+
+# list populations
+>>> spikes.get_populations_names()
+
+# open population
+>>> population = spikes['<name>']
+```
+
+### SpikePopulation
+
+```python
+# get all spikes [(node_id, timestep)]
+>>> population.get()
+[(5, 0.1), (2, 0.2), (3, 0.3), (2, 0.7), (3, 1.3)]
+
+# get all spikes betwen tstart and tstop
+>>> population.get(0.2, 1.0)
+[(2, 0.2), (3, 0.3), (2, 0.7)]
+
+# get spikes attribute sorting (by_time, by_id, none)
+>>> population.sorting
+'by_time'
+```
+
+### SomasReportReader
```

Comment:
```suggestion
### SomaReportReader
```
Didn't catch this when it was added to the code, but shouldn't this have been <singular>Report?

Reply:
What do you mean with Report? SomaReportReader?

Reply:
Ah, sorry, yes. Singular Soma + Report…

Reply:
Yes, I agree but I didn't see it either in the code and here I was consistent with it.

Reply:
I'm not sure that there's time to change this in code + documentation still, that's why I didn't request changes in the review…

More a grievance that this slipped my attention. These suggestions of mine don't have to (shouldn't) be acted on, I just wanted to highlight what I see as API misgivings.